package.json

```diff
@@ -27,18 +27,18 @@
         "@lwc/eslint-plugin-lwc": "^1.0.1",
         "@prettier/plugin-xml": "^0.13.1",
         "@sa11y/jest": "^0.3.1",
-        "@salesforce/eslint-config-lwc": "^2.0.0",
+        "@salesforce/eslint-config-lwc": "^2.2.0",
         "@salesforce/eslint-plugin-aura": "^2.0.0",
         "@salesforce/eslint-plugin-lightning": "^0.1.1",
         "@salesforce/sfdx-lwc-jest": "^0.14.0",
-        "eslint": "^7.28.0",
+        "eslint": "^7.32.0",
         "eslint-config-prettier": "^8.3.0",
-        "eslint-plugin-import": "^2.23.4",
-        "eslint-plugin-jest": "^24.3.6",
+        "eslint-plugin-import": "^2.24.2",
+        "eslint-plugin-jest": "^24.5.0",
         "husky": "^6.0.0",
```

Comment:
Some dependencies are not up to date in this list.
For example, Husky should be `7.0.2`.

Reply:
Everything is updated. Thank you for your review.